articles/virtual-desktop/app-attach-faq.md

```diff
@@ -55,6 +55,10 @@ Yes. MSIX app attach doesn't support auto-update for MSIX applications.
 
 All virtual machines (VMs) in a host pool that uses MSIX app attach must have read permissions on the file share where the MSIX images are stored. If it also uses Azure Files, they'll need to be granted both role-based access control (RBAC) and New Technology File System (NTFS) permissions.
 
+## Can I use Azure Active Directory Domain Services (AAD DS) with MSIX app attach?
```

Comment:
```suggestion
## Can I use Azure Active Directory Domain Services (AD DS) with MSIX app attach?
```

Reply:
AD DS is the wrong acronym, that is for traditional AD i.e. Active Directory Domain Services, which is not what this is about. This is talking about Azure Active Directory Domain Services hence AAD DS. These are two different things (albeit similar words/acronyms) but this needs to be completly accurate. Hence my comment is correct

Reply:
According to the style and branding guidelines, you can't abbreviate "Azure." It needs to be "Azure AD DS."

Reply:
Thats fine, but there are still two different products here: 1. Azure Active Directory Domain Servers (AAD DS) or (Azure AD DS),  and 2. Active Directory Domain Servers (AD DS). This FAQ is just talking about the first, so it needs to use the full acronym which seems like it needs to be Azure AD DS, but not (AD DS)

Reply:
So this needs to read as:
Can I use Azure Active Directory Domain Services (Azure AD DS) with MSIX app attach?

Reply:
Yes.

For the second one, say "Azure AD DS" and "Active Directory Domain Servers (AD DS)." Spelling out the non-Azure server name might help if the acronyms being too similar are confusing.